core/src/main/java/com/datastax/oss/driver/internal/core/context/DefaultDriverContext.java

```diff
@@ -314,7 +314,7 @@ protected EventBus buildEventBus() {
     return (Compressor<ByteBuf>)
         Reflection.buildFromConfig(
                 this,
-                DefaultDriverOption.PROTOCOL_COMPRESSOR_CLASS,
+                DefaultDriverOption.COMPRESSOR_CLASS,
```

Comment:
Another case where a section (`protocol`) was removed for one option, but not others.  Although this really makes sense as compression isn't really tied to the protocol from user's perspective.

Reply:
I moved up all "policies" to the top level.